parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -354,18 +360,31 @@ pub mod pallet {
 				payload.len() < T::MaxMessagePayloadSize::get() as usize,
 				SubmitError::MessageTooLarge
 			);
+
+			let base_fee = T::WeightToFee::weight_to_fee(
+				&T::WeightInfo::do_submit_message()
+					.saturating_add(T::WeightInfo::do_process_message())
+					.saturating_add(T::WeightInfo::do_commit_one_message()),
+			);
```

Comment:
So the base_fee has three components:

1. cost for submit message
2. cost for process message
3. cost for commit message

Reply:
Hmm, actually, I don't think the base fee should include the cost for submitting a message.

The extrinsics in the control pallet are all benchmarked, and so the weight of those extrinsics include the weight of submitting a message. So if we include the cost again, we're essentially double-billing users.

Its only process_message and commit message which we need to benchmark

Reply:
[Remove benchmark](https://github.com/Snowfork/snowbridge/pull/975/commits/3f685eae097d6b69b2295c78022000cf549040fd)